y_huff/config.json

```diff
@@ -8,16 +8,16 @@
     "GPL_CELL_PADDING": 4,
     "DPL_CELL_PADDING": 4,
     "FP_CORE_UTIL": 25,
+    "QUIT_ON_SYNTH_CHECKS": false,
```

Comment:
Can't we fix the errors?

Reply:
It's not worth the time investment.

Reply:
We should add a comment describing the errors and why were they waived.

Reply:
ok